### DISCUSSION TITLE
My model in C++ is slower than Python?

### DISCUSSION THREAD

**Discussion by Koruvika** [onnx/onnx#5866] [Q&A] [answered]
Model: FPN (Face Parsing)
Checkpoints: [FPN + ResNet-50](https://n-ws-620xz-pd11.s3pd11.sbercloud.ru/b-ws-620xz-pd11-jux/easyportrait/models/fpn.resnet50.512x512.pth)
OS: Windows 10
GPU: NVIDIA RTX 3050Ti 4GB

I converted it to ONNX format. When I timed it on Python using time.time(), it took approximately 17ms.
But when I timed it on C++ using std::chrono::high_resolution_clock, it took approximately 41ms.

I think the issue is from Windows. Does Windows limit ONNX model abilities?

**Comment by justinchuby** [comment, marked as answer]
Please feel free to create an issue in the [microsoft/onnxruntime](https://github.com/microsoft/onnxruntime/issues) repository. Be sure to include version numbers for relevant packages in your environment. Thanks!
